apps/condo/domains/miniapp/schema/B2BAppAccessRight.js

```diff
@@ -2,27 +2,56 @@
  * Generated by `createschema miniapp.B2BAppAccessRight 'user:Relationship:User:PROTECT;'`
  */
 
-const { Relationship } = require('@keystonejs/fields')
+const get = require('lodash/get')
 
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
 const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { getByCondition } = require('@open-condo/keystone/schema')
 const { GQLListSchema } = require('@open-condo/keystone/schema')
 
 const access = require('@condo/domains/miniapp/access/B2BAppAccessRight')
+const { ACCESS_RIGHT_SET_NOT_FOUND } = require('@condo/domains/miniapp/constants')
 const { SERVICE_USER_FIELD } = require('@condo/domains/miniapp/schema/fields/accessRight')
 
 
+const ERRORS = {
+    ACCESS_RIGHT_SET_NOT_FOUND: {
+        code: BAD_USER_INPUT,
+        type: ACCESS_RIGHT_SET_NOT_FOUND,
+        message: '"accessRightSet" must be connected to B2BApp, which specified in "app"',
+    },
+}
+
 const B2BAppAccessRight = new GQLListSchema('B2BAppAccessRight', {
     schemaDoc: 'Link between service user and B2B App. The existence of this connection means that this user has the rights to perform actions on behalf of the integration',
     fields: {
         user: SERVICE_USER_FIELD,
         app: {
             schemaDoc: 'Link to B2BApp.accessRights',
-            type: Relationship,
+            type: 'Relationship',
             ref: 'B2BApp.accessRights',
             isRequired: true,
             knexOptions: { isNotNullable: true }, // Required relationship only!
             kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
         },
+        accessRightSet: {
+            schemaDoc: 'Link to the set of access rights available to this application',
```

Comment:
Maybe available is not a good word right here...Since it's sound like an option to choose. Not like "this is the thing will be used to check your accesses" :)

Reply:
I'll write exactly as you suggested (`this is the thing will be used to check your accesses`) :)